collectors/VMStatsCollector.py

```diff
@@ -8,12 +8,14 @@ class VMStatsCollector(BaseCollector):
     def __init__(self):
         super().__init__()
         self.vrops_entity_name = 'virtualmachine'
+        self.rubric = os.environ['RUBRIC']
         self.wait_for_inventory_data()
         self.name = self.__class__.__name__
 
     def collect(self):
         gauges = self.generate_gauges('stats', self.name, self.vrops_entity_name,
-                                      [self.vrops_entity_name, 'vcenter', 'datacenter', 'vccluster', 'hostsystem', 'project'])
+                                      [self.vrops_entity_name, 'vcenter', 'datacenter', 'vccluster', 'hostsystem',
+                                       'project'], rubric=self.rubric)
```

Comment:
What happens, if it is not provided for a collector, because you want all rubrics, not just a single one?

Reply:
By default, `rubics=None`.  That means all metrics where considered. Look at the definition in `BaseCollector`.